src/python/WMComponent/DBS3Buffer/MySQL/DBSBufferFiles/Add.py

```diff
@@ -44,6 +41,7 @@ def execute(self, files = None, size = 0, events = 0, cksum = 0,
         binds = self.getBinds(files, size, events, cksum, datasetAlgo, status,
                               workflowID, inPhedex)
 
-        self.dbi.processData(self.sql, binds, conn=conn, transaction=transaction)
+        for sliceBinds in grouper(binds, 10000):
```

Comment:
I don't think this is necessary since we do slice in processData anyway. it would actually add a little more time.